search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -694,11 +697,17 @@
     :revision_date
     :result-format})
 
+(defn- filter-unrecognized-parameters
```

Comment:
This function is very similar to `unrecognized-params-validation` in `comm-app.services.search.parameter-validation`. I recommend putting this function in that namespace and calling it from there and here.